README.md

```diff
@@ -1,69 +1,84 @@
 # GOV.UK Notify - notifications-java-client
-Java client for GOV.UK Notify API
 
-Provides client calls, response marshalling and authentication for GOV.UK Notify API.
+This is the Java client for the GOV.UK Notify API.
 
-This application is built using java 8.
+It provides client calls, response marshalling and authentication for the GOV.UK Notify API.
+
+This application is built using Java 8.
 
 
 ## Usage
 
-Prior to usage a service must be created through the notify admin console. This will allow access to the API credentials you application, there are normal, test and team API keys available.
+Before usage, create a service through the Notify admin console. This provides access to the API credentials for your application. There are normal, test and team API keys available.
 Read the [API documentation](https://www.notifications.service.gov.uk/documentation) for more information.
 
-Add the maven dependency to your project.
+Add the Maven dependency to your project.
 mvn dependency to populate 
 
-**Import the NotificationClient**
-    `import uk.gov.service.notify.NotificationClient;`
-    `import uk.gov.service.notify.Notification;`
-    `import uk.gov.service.notify.NotificationList;`
-    `import uk.gov.service.notify.NotificationResponse;`
+**Import the `NotificationClient`**
+
+```
+import uk.gov.service.notify.NotificationClient;
+import uk.gov.service.notify.Notification;
+import uk.gov.service.notify.NotificationList;
+import uk.gov.service.notify.NotificationResponse;
+```
+
+**Create a new instance of `NotificationClient` and objects returned by the client**
+
+`NotificationClient client = new NotificationClient(secret, serviceId, "https://api.notifications.service.gov.uk");`
+
+**Send an email or text message**
 
-**Create a new instance of NotificationClient and objects returned by the client**
-    `NotificationClient client = new NotificationClient(secret, serviceId, "https://api.notifications.service.gov.uk");`
-    
-**Send and email or sms**
 `NotificationResponse response = client.sendEmail(templateId, emailAddress, personalisation);`
+
 or
+
 `NotificationResponse response client.sendSms(templateId, mobileNumber, personalisation);`
 
-* mobile-number is the mobile phone number for the notification
-    * Only UK mobiles are supported
-    * Must start with +44
-    * Must not have leading zero
-    * Must not have any whitespace, punctuation etc.
+* `mobileNumber` is the mobile phone number for the notification
+    * must be a UK mobile number
+    * must start with +44
+    * must not have a leading zero
+    * must not have any whitespaces or punctuation
     * valid format is +447777111222
```

Comment:
This isn’t true, we try to be as liberal as possible about which phone numbers we accept (see [a list of valid formats](https://github.com/alphagov/notifications-utils/blob/master/tests/test_recipient_validation.py#L14-L23)).